lightningd/invoice.c

```diff
@@ -1414,6 +1396,7 @@ static struct command_result *json_waitanyinvoice(struct command *cmd,
 				       " is non-trivial.");
 }
 
+
```

Comment:
It was to avoid line number clash with our autogen code; might be unnecessary at end of series though.